WalletWasabi/Wallets/Wallet.cs

```diff
@@ -400,7 +400,7 @@ private async Task LoadDummyMempoolAsync()
 						if (mempoolHashes.Contains(hash.ToString().Substring(0, compactness)))
 						{
 							txsToProcess.Add(tx);
-							Logger.LogInfo($"Transaction was successfully tested against the backend's mempool hashes: {hash}.");
+							Logger.LogInfo($"{WalletName}: Transaction was successfully tested against the backend's mempool hashes: {hash}.");
```

Comment:
Add apostrophes for consistency?

Reply:
I am not sure if it looks correct here because wallet name is the first word here and adding `:` helps in understanding that rest of the things mentioned in the log are for this wallet

Reply:
Yeah, can be.

The second option is to have all logger calls in format `Logger.LogInfo($"[{WalletName}] msg here");`. That might be more programmer friendly than user-friendly, I don't know, but personally I like it.

Reply:
The single quote is better - it is what we using elsewhere. 

Logger.LogInfo($"'{WalletName}' msg here");

Reply:
Done in https://github.com/zkSNACKs/WalletWasabi/pull/6940/commits/7b042600f46eafb3f66b1b0c47f718a5888959c6